spec/numberformat.html

```diff
@@ -30,9 +30,19 @@ <h1>SetNumberFormatDigitOptions ( _intlObj_, _options_, _mnfdDefault_, _mxfdDefa
           1. Set _intlObj_.[[MaximumSignificantDigits]] to _mxsd_.
         1. Else if _mnfd_ is not *undefined* or _mxfd_ is not *undefined*, then
           1. Set _intlObj_.[[RoundingType]] to ~fractionDigits~.
-          1. Let _mnfd_ be ? DefaultNumberOption(_mnfd_, 0, 20, _mnfdDefault_).
+          1. If _mxfd_ is *undefined*, then
```

Comment:
Does this work?

```
1. Let _mnfdActualDefault_ be ? min( _mnfdDefault_, DefaultNumberOption(_mxfd_, 0, 20, 20) ).
1. Let _mnfd_ be ? DefaultNumberOption(_mnfd_, 0, 20, _mnfdActualDefault_).
```

Keep the rest of the if statement as-is:

```
1. Let _mxfdActualDefault_ be max( _mnfd_, _mxfdDefault_ ).
1. Let _mxfd_ be ? DefaultNumberOption(_mxfd_, _mnfd_, 20, _mxfdActualDefault_).
1. Set _intlObj_.[[MinimumFractionDigits]] to _mnfd_.
1. Set _intlObj_.[[MaximumFractionDigits]] to _mxfd_.
```

Reply:
Verified that it works on V8, will push this nicer version.